include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -181,7 +181,35 @@ class concatenated_sequences
     using size_type = ranges::size_type_t<data_delimiters_type>;
     //!\}
 
-    /* rule of six */
+protected:
+    //!\cond
+    // unfortunately we cannot specialise the variable template so we have to add an auxiliary here
+    template <typename t>
+        requires (dimension_v<t> == dimension_v<value_type> + 1) &&
```

Comment:
what happens if the `value_type=int` ? related to the above question regarding the anchor of a basic type

Reply:
then the expression is invalid. This is a container-of-container (conceptionally), so everything that has value_type which is not range is not compatible.